element_calcium_imaging/imaging.py

```diff
@@ -152,8 +151,23 @@ def make(self, key):
             else:
                 raise NotImplementedError('Unknown method: {}'.format(method))
         elif task_mode == 'trigger':
-            raise NotImplementedError(f'Automatic triggering of {method} analysis'
-                                      f' is not yet supported')
+            method = (ProcessingTask * ProcessingParamSet * ProcessingMethod * scan.Scan & key).fetch1('processing_method') 
+            if method == 'suite2p':
+                import suite2p
+
+                suite_ops = (ProcessingTask * ProcessingParamSet & key).fetch1('params')
+                suite_ops['save_path0'] = (ProcessingTask & key).fetch1('processing_output_dir')
+
+                db = {}                
```

Comment:
PEP8: Use English words or phrases for variable names. No acronyms.

Reply:
I'm renaming `suite_ops` as `suite2p_params` and `db` as `suite2p_paths`. This change will be available in my next PR.